docs/theme/home.html

```diff
@@ -16,7 +16,7 @@
         Python, JavaScript, and Ruby) with a common output format, and developer
         tools that consume this format.
 
-        <p>srclib originated inside <a target="_blank" href="https://sourcegraph.com">Sourcegraph</a>, where it powers intelligent code search over hundreds of thousands of projects.
+        <p>srclib originated inside <a target="_blank" href="https://www.sourcegraph.com">Sourcegraph</a>, where it powers intelligent code search over hundreds of thousands of projects.
```

Comment:
It doesn't redirect right now, though. I don't have strong opinions on the `www` vs no-`www` question, but I think we should be consistent. Currently, all other URLs omit the `www`, so let's do the same here. #bikeshedding